Conduit/Conduit/Dto/UserD.cs

```diff
@@ -3,7 +3,6 @@
     public class UserD
     {
         public string UserName { get; set; }
-        public string Email { get; set; }
```

Comment:
can you clarify here, why you remove Email prop?

Reply:
not sure why but maybe by mistake